### ISSUE TITLE
JavaScript widget classes must be created for each Event subclass

### ISSUE THREAD

**Issue by colinsullivan** [concertsoundorganizer/Concert#46]
Widgets are awesome

Prerequisites: #43, #44, #45

**Comment by colinsullivan** [comment]
Partially completed in 324be95eb51ca52dcf4eabb4c7289102957fb148

**Comment by colinsullivan** [comment]
No longer needed, using the base Event model for all events.
